LeapEngine/SceneGraph/GameObject.h

```diff
@@ -15,6 +17,14 @@ namespace leap
 
 	class GameObject final
 	{
+		struct ComponentInfo
+		{
+			std::unique_ptr<Component> pComponent;
+			uint32_t ID;
```

Comment:
Use unsigned instead of typedef uint32_t

Reply:
Why not the typedef? Makes the code shorter and more clear with what intended size is intended instead of the ambigious `int`